appium-cookies/lib/helpers.js

```diff
@@ -0,0 +1,20 @@
+
+import _ from 'underscore';
+
+export default class Helpers {
```

Comment:
This being a class seems a bit heavyweight to me. If it is just a couple of useful functions with no state or anything, they could just be functions that get exported.